### ISSUE TITLE
High Priority: Municipality Computing data issue 

### ISSUE THREAD

**Issue by cemachelen** [cemac/AgroClimatic-Monitor#36]
https://agclimatebr.leeds.ac.uk/en/individual/2611101/  and others just say computing and never process

**Comment by cemachelen** [comment]
Dan set this up to have a mix of preprocessed and on the fly processing. There was issues with the on the fly processing which I have partially fixed on a local branch that should be able to test and implement. Dans notes suggested on the fly calculations were fast but I'll also look at just preprocessing the lot

**Comment by Murilodsv** [comment]
Thanks a lot, and please stick to the more realistic solution given the deadline.

**Comment by cemachelen** [comment]
part working now. H5 files don't contain all municipalities so some still missing.

**Comment by Murilodsv** [comment]
Thanks! does it relates to how h5 files are being created or its an issue with the cities' shapefile? I can provide another shapefile if its needed.

**Comment by cemachelen** [comment]
I can't quite track down why some municipalities have no datasets. The shape files look fine and now the h5 files have entries for every municipalities.

**Comment by Murilodsv** [comment]
Do you think if we provide preprocessed csv files for each municipality with the indexes, you could convert them into json and place in https://github.com/cemac/AgroClimatic-Monitor/tree/agr/processed/muncipalities until the release on Feb-1?

**Comment by cemachelen** [comment]
@Murilodsv yes that could work I'll try have another little look today if I can figure out what stopping all the datasets from being incorporated into the h5.

**Comment by Murilodsv** [comment]
Hi @cemachelen I just see that many individual municipalities are now working. For some, the bottom plot is still empty (e.g. Vinhedo), even though the neighbor city has indices (e.g. Valinhos). It's not critical but it should be investigated.

Also, the "generate report" button is currently sending the user to the homepage, whereas it was planned to be a download button for a csv file containing the indexes for the corresponding city. Example with dummy data:

- [3538709_indices_report.csv](https://github.com/cemac/AgroClimatic-Monitor/files/7963680/3538709_indices_report.csv)

If fixing the button it's too much for the release date, let us remove it and add it afterwards. Because there are other issues that also need attention before releasing.
